ros-one.repos

```diff
@@ -341,28 +349,32 @@ repositories:
     version: master
   jsk_common:
     type: git
-    url: https://github.com/ros-o/jsk_common.git
-    version: obese-devel
+    url: https://github.com/mqcmd196/jsk_common.git
+    version: techfak-ros-o
   jsk_common_msgs:
     type: git
     url: https://github.com/jsk-ros-pkg/jsk_common_msgs.git
     version: master
-  jsk_recognition:
-    type: git
-    url: https://github.com/jsk-ros-pkg/jsk_recognition.git
-    version: master
+  jsk_model_tools:
+    type: git
+    url: https://github.com/mqcmd196/jsk_model_tools.git
+    version: techfak-ros-o
+  # jsk_pr2eus:
+  #   type: git
+  #   url: https://github.com/jsk-ros-pkg/jsk_pr2eus.git
+  #   version: master
+  # jsk_recognition:
+  #   type: git
+  #   url: https://github.com/jsk-ros-pkg/jsk_recognition.git
+  #   version: master
```

Comment:
Why do you remove this?

Reply:
This package requires jsk_data in jsk_common, so it fails. Adding to known_failures like https://github.com/ubi-agni/ros-builder-action/blob/5cb28dc920c82a9208ef3e63e7f87f31298b2206/ros-one.repos#L1076 is fine?

Reply:
But jsk_common is built as well. So, jsk_data should be available, shouldn't it?

Reply:
https://github.com/ubi-agni/ros-builder-action/pull/36/commits/d8f5881d0c1b958c4fc9b9a9677baa2e1e38faa6